platform/ios/src/MGLMapView.mm

```diff
@@ -564,6 +565,8 @@ - (void)commonInit
     if ([UIApplication sharedApplication].applicationState != UIApplicationStateBackground) {
         [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
     }
+
+    [MGLSDKUpdateChecker checkForUpdates];
```

Comment:
(Responded initially without refreshing and seeing @boundsj comments.)

Sure, we can move this to a new `+load` or `+initialize` in MGLMapView. This update check is clearly not essential, so that’s why I dropped it at the end of `-commonInit`.